### DISCUSSION TITLE
v2.2

### DISCUSSION THREAD

**Discussion by sbrl** [sbrl/Pepperminty-Wiki-Client-Android#15] [Announcements]
It's the release you've all been waiting for - and it's finally here! This release contains 2 *highly* requested features:

 - Page editing(!!!!!)
 - Dark theme

It took an enormous amount of effort, but I got there in the end :D

Known issue: the dark theme only follows your system theme at this time - the setting doesn't do anything yet because I haven't figured out how to make it work (please open an issue if you have a solution :D

<hr /><em>This discussion was created from the release <a href='https://github.com/sbrl/Pepperminty-Wiki-Client-Android/releases/tag/v2.2'>v2.2</a>.</em>

**Comment by virtadpt** [comment]
https://drwho.virtadpt.net/graphics/ani-john_dancing.gif
